WooCommerce/Classes/ViewRelated/Orders/Payment Methods/PaymentMethodsViewModel.swift

```diff
@@ -245,9 +245,10 @@ final class PaymentMethodsViewModel: ObservableObject {
                                                                        rootViewController: rootViewController,
                                                                        alertsPresenter: alertsPresenter,
                                                                        onboardingPresenter: self.cardPresentPaymentsOnboardingPresenter,
+                                                                       tapToPayAlertProvider: BuiltInReaderConnectionAlertsProvider(),
```

Comment:
does it make sense to keep `tapToPayAlertsProvider` as before to minimize the changes?
```suggestion
                                                                       tapToPayAlertProvider: tapToPayAlertsProvider,
```